### PR TITLE
add WordNet API

### PR DESCRIPTION
**Pull request by wmshort**

### CODE REVIEW

**Comment by kylepjohnson** [comment]
Got it. FYI @todd-cook this will be the PR we'll work from

**Comment by kylepjohnson** [comment]
Previous comments: https://github.com/cltk/cltkv1/pull/62

File: src/cltkv1/wordnet/wordnet.py [hunk 1]
```diff
@@ -0,0 +1,2288 @@
...
+    def __init__(
+        self,
+        iso_code,
+        root=None,
+        fileids=None,
+    ):
+        if not root:
+            iso_map ={'lat': 'latin', 'grk': 'greek', 'skt': 'sanskrit'}
+            root = os.path.join(
+            get_cltk_data_dir(), f"{iso_map[iso_code]}/model/{iso_map[iso_code]}_models_cltk/semantics/wordnet/"
```
Comment (kylepjohnson) [inline src/cltkv1/wordnet/wordnet.py:2070]: I see what went wrong. In my doctest for ``, I create a custom OS variable (`$CLTK_DATA`) but do not delete it. Will add this now to master, then I'll update this branch against the new master.

https://github.com/cltk/cltkv1/blob/9f66f72fcc64a6b42a5acebbdaa093b1cd5cca9a/src/cltkv1/utils/utils.py#L115

Comment (kylepjohnson) [reply src/cltkv1/wordnet/wordnet.py:2070]: Fixed here https://github.com/cltk/cltkv1/commit/d5ea3693336a1dda330894ec7e5c9e7172d7d5ae

**Comment by kylepjohnson** [comment]
@todd-cook How do we update Bill's branch with my little fix to master?

**Comment by todd-cook** [comment]
> @todd-cook How do we update Bill's branch with my little fix to master?

I've taken the liberty of pushing changes to his branch to iron out the tests. Not the best way, but opening a PR to a PR is a bit pedantic and certainly creates delays. With the dir fix, we should be close.

**Comment by kylepjohnson** [comment]
@todd-cook I am fine to merge this. All of the wordnet tests pass.

(2 broken tests that are due to an upstream change in Stanza for Coptic.)

**Pull request merged**